web/packages/design/src/StepSlider/StepSlider.tsx

```diff
@@ -17,7 +17,7 @@ import React, { useState, useRef, useEffect, useCallback } from 'react';
 import { TransitionGroup, CSSTransition } from 'react-transition-group';
 import styled from 'styled-components';
 
-import { Box } from 'design';
+import Box from 'design/Box';
```

Comment:
curious why change?

Reply:
Yeah it causes a weird circular dependency issue